install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -82,26 +85,13 @@ kubecostAggregator:
     # ensure disk is specd high enough, and check for bottlenecks
     # default: 128Gi
     storageRequest: 512Gi
-federatedETL:
-  federatedCluster: true
-kubecostModel:
-  containerStatsEnabled: true
-  federatedStorageConfigSecret: federated-store
-kubecostProductConfigs:
-  clusterName: YOUR_CLUSTER_NAME
-  cloudIntegrationSecret: cloud-integration
-  productKey:
-    enabled: true
-    key: YOUR_KEY
-prometheus:
-  server:
-    global:
-      external_labels:
-        cluster_id: YOUR_CLUSTER_NAME
```

Comment:
Some of these are pretty useful to be aware of; why remove them? Just to focus on Aggregator config?